src/goto-cc/gcc_mode.h

```diff
@@ -18,6 +18,8 @@ Date: June 2006
 
 #include "goto_cc_mode.h"
 
+#include <set>
```

Comment:
No use of `std::set` in this file?!

Reply:
const std::map<std::string, std::set<std::string>> arch_map;